dev/com.ibm.websphere.org.eclipse.microprofile.rest.client.1.0/bnd.bnd

```diff
@@ -17,11 +17,11 @@ javac.target: 1.8
 Bundle-SymbolicName: com.ibm.websphere.org.eclipse.microprofile.rest.client.1.0; singleton:=true
 
 Export-Package: \
-  org.eclipse.microprofile.rest.client;version=1.0,\
-  org.eclipse.microprofile.rest.client.annotation;version=1.0,\
-  org.eclipse.microprofile.rest.client.ext;version=1.0,\
-  org.eclipse.microprofile.rest.client.inject;version=1.0,\
-  org.eclipse.microprofile.rest.client.spi;version=1.0
+  org.eclipse.microprofile.rest.client;version=1.0.1,\
```

Comment:
Why did you update the package version? Did you update any classes in this package? To me, it sounds like you use the bundle version for package version, which is incorrect.